lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -115,9 +129,13 @@ def convert_username(value: str) -> str:
         internal username representation
     """
     try:
-        return Auth.validate_user(value)
+        user, verified = Auth().verify_user(value)
     except Exception:
         raise ConversionError(value, "username", type(value).__name__)
+    if verified or (user and user.is_admin()):
+        return value
```

Comment:
You've changed the semantics of the conversion; `validate_user` returns `str(user.id)`, which is the Elasticsearch owner key, __not__ the username.